packages/grid/src/vaadin-grid-column.js

```diff
@@ -845,6 +845,14 @@ class GridColumn extends ColumnBaseMixin(DirMixin(PolymerElement)) {
         value: false,
       },
 
+      /**
+       * Custom function for rendering the header content.
```

Comment:
`rowHeader` isn't a function.
The documentation should be aligned with the type definitions.

Reply:
Thanks, indeed, not much info in the documentation, I am updating it with more info, insights.